domains/donnietisdale.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "donnietisdale",
+        "email": "donnie@donnietisdale.com"
+    },
+    "record": {
+        "TXT": "v=spf1 include:_spf-us.ionos.com ~all"
```

Comment:
TXT records alone are used for verification only, you need to have other records for this.